factory/etc/systemd/bootchart.conf

```diff
@@ -0,0 +1,8 @@
+[Bootchart]
+Samples=36000
+Frequency=20
+Relative=yes
+Filter=no
+# Memory usage produces a bad overlay in the svg
```

Comment:
Sadness, is this something that needs to be fixed upstream? or like make it generate memory usage separately somewhere?